src/webhook/mutation/pod_mutator/oneagent_mutation/containers_test.go

```diff
@@ -71,7 +71,7 @@ func TestMutateUserContainers(t *testing.T) {
 		assert.Equal(t, request.Pod.Spec.Containers[0].Image, request.InstallContainer.Env[1].Value)
 
 		assert.Len(t, request.Pod.Spec.Containers[0].VolumeMounts, initialContainerVolumeMountsLen+5)
-		assert.Len(t, request.Pod.Spec.Containers[0].Env, initialNumberOfContainerEnvsLen+4)
+		assert.Len(t, request.Pod.Spec.Containers[0].Env, initialNumberOfContainerEnvsLen+6)
```

Comment:
nitpick: 6 could be refactored into a constant since it is used multiple times and so that it has a name